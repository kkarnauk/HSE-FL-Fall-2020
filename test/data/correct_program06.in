f :- first, second; third, extra (hello ((world))) wot, he.
