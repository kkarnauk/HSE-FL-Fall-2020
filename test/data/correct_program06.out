|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|disjunction
|	|	|	|conjunction
|	|	|	|	|identifier = 'first'
|	|	|	|	|identifier = 'second'
|	|	|	|conjunction
|	|	|	|	|identifier = 'third'
|	|	|	|	|conjunction
|	|	|	|	|	|atom
|	|	|	|	|	|	|identifier = 'extra'
|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|identifier = 'hello'
|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|identifier = 'world'
|	|	|	|	|	|	|identifier = 'wot'
|	|	|	|	|	|identifier = 'he'